### PR TITLE
Update all packages

### PR DESCRIPTION
**Pull request by flppv**
When installing `react-top-loading-bar` package npm complains on incompatible package versions. I updated all packages to a freshest major versions in this PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,32 +28,32 @@
     "deploy": "gh-pages -d example/build"
   },
   "peerDependencies": {
-    "react": "^16.0.0"
+    "react": "^17.0.2"
```
Comment (michielvanderros-tomtom) [inline package.json:31]: As suggested in issue #49 it may be best to allow compatibility with both 16 and 17.
```suggestion
    "react": "^16 || ^17"
```

**Review by michielvanderros-tomtom — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "react-top-loading-bar",
+  "name": "@onomyprotocol/react-top-loading-bar",
```
Comment (klendi) [inline package.json:2]: Package name is changed, please revert to original package name

File: package.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
     "node": ">=10"
   },
   "scripts": {
-    "build": "microbundle-crl",
+    "build": "react-scripts build",
```
Comment (klendi) [inline package.json:19]: Why have you changed to react-scripts ?

File: package.json [hunk 4]
```diff
@@ -83,9 +83,8 @@
     "top loading bar",
     "component"
   ],
-  "homepage": "https://klendi.github.io/react-top-loading-bar",
   "repository": {
     "type": "git",
-    "url": "https://github.com/klendi/react-top-loading-bar.git"
+    "url": "https://github.com/onomyprotocol/react-top-loading-bar.git"
```
Comment (klendi) [inline package.json:88]: Also in here change back the package url to the original one

File: package.json (hunk 2 above)
Comment (michielvanderros-tomtom) [inline package.json:2]: Would this change the name of the main package into the forker's?

**Review by klendi — CHANGES_REQUESTED** [review]

**Comment by flppv** [comment]
Whoops, sorry, will fix

**Pull request closed without merging**